code/backend/batch/utilities/orchestrator/SemanticKernel.py

```diff
@@ -0,0 +1,137 @@
+import json
+import logging
+
+from semantic_kernel import Kernel
+from semantic_kernel.connectors.ai.open_ai.utils import get_tool_call_object
+from semantic_kernel.contents import ChatHistory
+
+from ..common.Answer import Answer
+from ..helpers.LLMHelper import LLMHelper
+from ..plugins.ChatPlugin import ChatPlugin
+from ..plugins.PostAnsweringPlugin import PostAnsweringPlugin
+from .OrchestratorBase import OrchestratorBase
+
+logger = logging.getLogger(__name__)
+
+
+class SemanticKernelOrchestrator(OrchestratorBase):
+    def __init__(self) -> None:
+        super().__init__()
+        self.kernel = Kernel()
+        self.llm_helper = LLMHelper()
+
+        # Add the Azure OpenAI service to the kernel
+        self.chat_service = self.llm_helper.get_sk_chat_completion_service("aoai")
```

Comment:
nit: From my early investigation, this string can be anything - do you think we should disambiguate from aoai and make it something like "cwyd" so people understand it could be anything. Also feel free to ignore!

Reply:
Sure, updated